tp/tp.py

```diff
@@ -147,6 +148,10 @@ def guess_target(self):
             if candidate != previous:
                 self.logger.debug(">> guess_target(): changed target from %s to %s", previous, candidate)
                 self.target = candidate
+        else:
+            self.logger.info("guess_target(): not updating target for instances, waiting for cooldown!")
```

Comment:
If you always log this as `INFO`, it doesn't make sense to log the following line in `DEBUG`. Change the following line to `INFO` and adjust the string message.